frontend/.vscode/extensions.json

```diff
@@ -0,0 +1,7 @@
+{
+  "recommendations": [
```

Comment:
a few more I use:

- "mgmcdermott.vscode-language-babel"
- "stylelint.vscode-stylelint"

Reply:
I also use a bunch of snippets extensions, but it's more of a preference than a necessity..